modules/virt-importing-vm-to-block-pv.adoc

```diff
@@ -4,78 +4,76 @@
 
 :_content-type: PROCEDURE
 [id="virt-importing-vm-to-block-pv_{context}"]
-= Importing a virtual machine image to a block persistent volume using data volumes
+= Importing a virtual machine image into block storage by using a data volume
 
-You can import an existing virtual machine image into your {product-title} cluster. {VirtProductName} uses data volumes to automate the importing data and the creation of an underlying persistent volume claim (PVC). You can then reference the data volume in a virtual machine manifest.
+You can import a virtual machine image into block storage by using a data volume. You reference the data volume in a `VirtualMachine` manifest before you create a virtual machine.
 
 .Prerequisites
 
-* A virtual machine disk image, in RAW, ISO, or QCOW2 format, optionally compressed by using `xz` or `gz`.
-* An `HTTP` or `s3` endpoint where the image is hosted, along with any authentication credentials needed to access the data source
-* At least one available block PV.
+* A virtual machine disk image in RAW, ISO, or QCOW2 format, optionally compressed by using `xz` or `gz`.
+* An HTTP or HTTPS endpoint where the image is hosted, along with any authentication credentials needed to access the data source.
 
 .Procedure
 
-. If your data source requires authentication credentials, edit the `endpoint-secret.yaml` file, and apply the updated configuration to the cluster.
+. If your data source requires authentication, add the credentials to your cluster:
 
-.. Edit the `endpoint-secret.yaml` file with your preferred text editor:
+.. Create a `Secret` manifest, specifying the data source credentials, and save it as `endpoint-secret.yaml`:
```

Comment:
Since this is prescribing the name of the file, you can use `endpoint-secret` and `endpoint-secret.yaml` without brackets throughout. I think either way is fine, but I'd pick one for this whole PR and try to stick to it.